terraform/templates/ec2_user_data.sh

```diff
@@ -25,7 +25,7 @@ yum install -y /tmp/node_exporter.rpm
 systemctl start node_exporter
 
 cat > /etc/cron.d/metric_collector <<"EOF"
-* * * * * root   docker container ls -q --filter label=vcs-upstream | xargs docker inspect --format='{{.State.StartedAt}}' | xargs date +"\%s" -d | xargs echo "ecs_start_time_seconds " > /var/lib/node_exporter/textfile_collector/ecs_stats.prom
+* * * * * root   docker container ls -q --filter label=vcs-upstream --filter name=^ecs-dev-validator-.*-validator-.*$ | xargs docker inspect --format='{{.State.StartedAt}}' | xargs date +"\%s" -d | xargs echo "ecs_start_time_seconds " > /var/lib/node_exporter/textfile_collector/ecs_stats.prom
```

Comment:
This need to be generic, fullnode and faucet also uses this script to get uptime. if you filter on the name "validator", we will lose other metrics